src/main/java/com/iotiq/application/messages/cart/CartUpdateRequest.java

```diff
@@ -0,0 +1,13 @@
+package com.iotiq.application.messages.cart;
+
+import com.iotiq.application.domain.CartItem;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+
+@Getter
+@Setter
+public class CartUpdateRequest {
+    private List<CartItem> cartItems;
```

Comment:
For CartItem: This should not be an entity itself. It should be a separate record (the representation of the CartItem ) even if it is the same for now.